fbw-a32nx/src/wasm/fadec_a320/src/FadecGauge.h

```diff
@@ -79,6 +82,35 @@ class FadecGauge {
     return false;
   }
 
+  bool addInputDataDefinition(const HANDLE connectionHandle,
+                              const SIMCONNECT_DATA_DEFINITION_ID groupId,
+                              const SIMCONNECT_CLIENT_EVENT_ID eventId,
+                              const std::string& eventName,
+                              const bool maskEvent) {
+    HRESULT result = SimConnect_MapClientEventToSimEvent(connectionHandle, eventId, eventName.c_str());
+
+    if (result != S_OK) {
+      // failed -> abort
+      return false;
+    }
+
+    result = SimConnect_AddClientEventToNotificationGroup(connectionHandle, groupId, eventId, maskEvent ? TRUE : FALSE);
+    if (result != S_OK) {
+      // failed -> abort
+      return false;
+    }
+
+    result = SimConnect_SetNotificationGroupPriority(connectionHandle, groupId, SIMCONNECT_GROUP_PRIORITY_HIGHEST_MASKABLE);
```

Comment:
https://docs.flightsimulator.com/html/Programming_Tools/SimConnect/SimConnect_API_Reference.htm?rhhlterm=simconnect_group_priority_highest_maskable&rhsearch=SIMCONNECT_GROUP_PRIORITY_HIGHEST_MASKABLE

Seems to be deprecated.

Reply:
Yes though we use it copiously in the FBW c++ code so I wanted to keep it similar

Reply:
Ok, let's keep it this way. We can refactor it separately and also take care of the other occurrences then 👍